dll/win32/kernel32/client/sysinfo.c

```diff
@@ -413,6 +413,30 @@ GetNumaAvailableMemoryNode(IN UCHAR Node,
     return TRUE;
 }
 
+/*
+ * @implemented
+ */
+BOOL
+WINAPI
+GetPhysicallyInstalledSystemMemory(PULONGLONG TotalMemoryInKilobytes)
+{
+    if (TotalMemoryInKilobytes == NULL)
+    {
+        SetLastError(ERROR_INVALID_PARAMETER);
+        return FALSE;
+    }
+
+    MEMORYSTATUSEX status;
+    status.dwLength = sizeof(status);
+
+    if (!GlobalMemoryStatusEx( &status )) {
```

Comment:
This is not our standard code style. Where did this come from?

Reply:
I apologize, I had not yet come across https://reactos.org/wiki/Coding_Style.